xml/snmp_sysdescr.xml

```diff
@@ -749,6 +749,21 @@
     <param pos="0" name="os.device" value="WAP"/>
   </fingerprint>
 
+  <!--======================================================================
+                               Aruba
+  =======================================================================-->
+
+  <fingerprint pattern="^Aruba\s(JL\d+A)\s(\d+[A-Z]?)\S+\sSwitch.+ROM\s([A-Z]+(?:\.\d+)+)">
+    <description>HP Aruba Network Switch</description>
+    <example hw.product="JL256A" hw.family="2930F" hw.version="WC.16.01.0010">Aruba JL256A 2930F-48G-PoE+-4SFP+ Switch, revision WC.16.11.0004, ROM WC.16.01.0010</example>
+    <param pos="0" name="hw.device" value="Switch"/>
+    <param pos="0" name="hw.vendor" value="Aruba Networks"/>
+    <param pos="1" name="hw.product"/>
+    <param pos="2" name="hw.family"/>
+    <param pos="3" name="hw.version"/>
+    <param pos="0" name="os.vendor" value="Aruba Networks"/>
```

Comment:
Alot of switches tend to have an OS specific to them, have you checked to see if that is the case with these arubas switches? just if it is we can infer a os.product from it too.

Reply:
I can’t find Aruba documentation on this matter, might be better to play it safe and leave it out